BuildProject/scripts/pretest.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 build_ios_app() {
   echo "$0: [iOS] Building archive\n"
-  xcodebuild -workspace ios/BuildProject.xcworkspace -scheme BuildProject -archivePath build/ReactNativeMetricsBuildProject archive -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 11'
+  xcodebuild -workspace ios/BuildProject.xcworkspace -scheme BuildProject -archivePath build/ReactNativeMetricsBuildProject archive -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 14 Pro Max'
```

Comment:
Try adding `-allowProvisioningUpdates` to this command line to resolve the error you see.

The error message you encountered could have been something new in Xcode 14. My memory is fuzzy, but either:

1. Earlier versions of Xcode didn't require you to sign archives built for simulator. They only required you to sign archives built for device. Or,
2. Earlier versions of Xcode were more relaxed in their willingness to update your provisioning profile.

Reply:
Cool.  I'm hitting a different error now.  I'll play with it.

```
/Users/danielhill/promotedai/react-native-metrics/BuildProject/ios/BuildProject.xcodeproj: error: There are no accounts registered with Xcode. Add your developer account to Xcode (in target 'BuildProject' from project 'BuildProject')
/Users/danielhill/promotedai/react-native-metrics/BuildProject/ios/BuildProject.xcodeproj: error: No profiles for 'ai.promoted.ReactNativeMetricsBuildProject' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'ai.promoted.ReactNativeMetricsBuildProject'. (in target 'BuildProject' from project 'BuildProject')
```